src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1681,6 +1681,19 @@ protected override bool UpdateValueCore(UpdateValueParams updateValueParams)
                     NickName = value;
                     return true;
 
+                case "Position":
```

Comment:
Ok. But provide some comments here. If you say value.split(;), what is value here? give some example around this.